src/components/Layout/HomeContent.js

```diff
@@ -658,35 +666,35 @@ const reactConf2019Cover = '/images/home/conf2019/cover.svg';
 const communityImages = [
   {
     src: '/images/home/community/react_conf_fun.webp',
-    alt: 'People singing karaoke at React Conf',
+    alt: 'Pessoas cantando karaokê no React Conf',
   },
   {
     src: '/images/home/community/react_india_sunil.webp',
-    alt: 'Sunil Pai speaking at React India',
+    alt: 'Sunil Pai falando no React Índia',
```

Comment:
Acredito que não há necessidade do acento quando se trata do **nome do evento**.
```suggestion
    alt: 'Sunil Pai falando no React India',
```